libs/ui/src/lib/facet/list/list.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core'
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+import { FormsModule } from '@angular/forms'
+
+import { ListComponent } from './list.component'
+
+describe('ListComponent', () => {
+  let component: ListComponent
+  let fixture: ComponentFixture<ListComponent>
+
+  const modelMock = {
+    key: 'tag',
```

Comment:
We should declare a `Typescript` definition

Reply:
I agree, but doesn't it belong in lib-search ?

Reply:
If you put the model in `lib-search`, then `lib-ui` will depend on `lib-search` and we'll have a cycle dependency.